src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,32 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { Todo } from '../../types/Todo';
+import { User } from '../../types/User';
+import { getUser } from '../../api';
+
+type Props = {
+  visibleModal: Todo,
+  setVisibleModal: (todo: Todo | null) => void;
+};
+
+export const TodoModal: React.FC<Props> = ({
+  visibleModal, setVisibleModal,
+}) => {
+  const [user, setUser] = useState<User | null>(null);
+  const [loading, setLoading] = useState(true);
+
+  useEffect(() => {
+    getUser(visibleModal.userId)
+      .then(setUser)
+      .catch(error => new Error(error.message))
```

Comment:
Error is being created but not thrown or logged, this is anti-pattern. It won't provide any feedback during runtime. Consider logging the error or throwing it